src/prismo/pl/_plotting.py

```diff
@@ -1008,13 +1001,18 @@ def _prettify_factor_names(
     """Prettify factor names.
 
     Replace underscores with spaces. If `replacements` is not empty, replace the keys with the values.
-    Shorten the labels to `abbreviation_length` characters.
+    Shorten the labels to `abbreviation_length` characters. If `end_with_database` is `True`,
+    put the database name in square brackets at the end of the label.
+
+    Example:
+    `Reactome ABC` becomes `ABC [R]`, if `replacements` is `{"Reactome": "R"}` and `end_with_database` is `True`.
```

Comment:
Is that correct? If I'm reading the code right `replacements` should be `{"Reactome": "[R]"}` for this to work.

Reply:
Good catch, thanks!